pkg/operator/controller/ingress/deployment.go

```diff
@@ -336,7 +341,49 @@ func desiredRouterDeployment(ci *operatorv1.IngressController, ingressController
 	secretName := controller.RouterEffectiveDefaultCertificateSecretName(ci, deployment.Namespace)
 	deployment.Spec.Template.Spec.Volumes[0].Secret.SecretName = secretName.Name
 
+	var (
+		accessLogging  *operatorv1.AccessLogging
+		accessLogLevel = "debug"
+	)
 	if enabled, level := ExtraLoggingEnabled(ci, ingressConfig); enabled {
```

Comment:
What if instead we delete that annotation and all the code to implement it? It was never supported.

Reply:
All right, I can do that.

Reply:
Deleted.  This led me to notice another problem:  The logic to create the configmap for rsyslog only looked at the annotation and not at `spec.logging.access`.  I have fixed this problem as well now.

Reply:
I pushed another changes that further tweaks that logic (it should only create the configmap if a container logging is used) and adds an end-to-end test that creates an ingress controller that uses container logging (if no configmap is created, the ingress controller pod should fail to start).